docs/Releasing.md

```diff
@@ -27,4 +27,171 @@ After checking that everything looks good, you just need to push the changes:
 
 ```
 git push --tags
```

Comment:
Should we keep this information above? I don't think the version command even works now...

Reply:
`yarn version` still works. It's one of the steps in the checklist even. With that said, I don't think anyone is bumping the version outside of the release process, so I removed the portion that references that in 9e5ecf1. I left the portion that just discusses creating a bundle though. I don't feel strongly about keeping it, but it seemed like it was correct, so I left it in. But let me know what you think.